pkg/bucketeer/api/api.go

```diff
@@ -254,10 +236,9 @@ func (c *client) sendHTTPRequest(url string, body interface{}) (*successResponse
 	if resp.StatusCode != http.StatusOK {
 		return nil, NewErrStatus(resp.StatusCode)
 	}
-	var sr successResponse
-	err = json.NewDecoder(resp.Body).Decode(&sr)
+	data, err := ioutil.ReadAll(resp.Body)
```

Comment:
```suggestion
	data, err := io.ReadAll(resp.Body)
```

Because ioutil packages is deprecated.
https://pkg.go.dev/io/ioutil
```Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code.```